kube-fluentd-operator.advisories.yaml

```diff
@@ -7,3 +7,8 @@ advisories:
       status: not_affected
       justification: component_not_present
       impact: The CVE was against an older version of ruby itself which included this in the stdlib.
+
+  CVE-2023-36617:
+    - timestamp: 2023-07-07T12:19:11.120734-04:00
```

Comment:
looks like the date might be a copy / paste

Reply:
fixed